tests/test_api.py

```diff
@@ -23,9 +22,10 @@
 
 from . import base
 
+# To mock the correct exception when testion on Python 2 and 3, store the
+# appropriate exception for the current Python version in SSL_ERROR.
 if six.PY3:
     import ssl
-    print("imported ssl: {}".format(ssl))
     SSL_ERROR = ssl.SSLError
```

Comment:
Is client code going to run into this issue? Should we expose an SSLError in sgsix?

Reply:
Create a new variable in sgsix, `ShotgunSSLError` that contains the appropriate SSLError class for the running Python version, and updated test_api to use that variable instead. ✅